collector/accelerators.go

```diff
@@ -84,6 +84,14 @@ var (
 		"0x2233": "NVIDIA RTX A5500",
 		"0x1e30": "NVIDIA RTX 8000 passive",
 		"0x2531": "NVIDIA RTX A2000",
+		"0x20b0": "NVIDIA A100 SXM4 40G",
```

Comment:
mabye
`"0x20b0": "NVIDIA A100 SXM4 40GB",`

Reply:
@juzhao, this seems like a very minor issue in description. I won't fixed here, but i will fix it in the future versions